ASM/src/config.asm

```diff
@@ -254,6 +256,9 @@ TRIFORCE_HUNT_ENABLED:
 TRIFORCE_PIECES_REQUIRED:
 .halfword 0xffff
 
+EPONAS_SONG_NOTES:
```

Comment:
This is a breaking change to the auto-tracker context which requires incrementing `AUTO_TRACKER_VERSION` and documenting in `Notes/auto-tracker-ctx.md`. To avoid that, move this above the `AUTO_TRACKER_CONTEXT` label.